.github/actions/tune-runner-vm/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Tune Runner VM
+description: |
+  Fine tunes the GitHub Runner VM operation system for Trino builds. 
```

Comment:
I think we should replace Trino here to Phoenix-Connectors

Reply:
Thank you.
I have removed the mention of Trino.